lifecycle/createfromartifacts.go

```diff
@@ -0,0 +1,55 @@
+package lifecycle
+
+import (
+	"errors"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	rtServicesUtils "github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	"github.com/jfrog/jfrog-client-go/lifecycle"
+	"github.com/jfrog/jfrog-client-go/lifecycle/services"
+	"github.com/jfrog/jfrog-client-go/utils/io/content"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"path"
+)
+
+func (rbc *ReleaseBundleCreateCommand) createFromArtifacts(lcServicesManager *lifecycle.LifecycleServicesManager,
+	rbDetails services.ReleaseBundleDetails, queryParams services.CommonOptionalQueryParams) (err error) {
+
+	rtServicesManager, err := utils.CreateServiceManager(rbc.serverDetails, 3, 0, false)
+	if err != nil {
+		return err
+	}
+
+	log.Info("Searching artifacts...")
+	searchResults, callbackFunc, err := utils.SearchFiles(rtServicesManager, rbc.spec)
```

Comment:
You don't need to search files. This is not efficient.
Instead, I'd recommend converting the pattern to AQL and using createFromAql.

Reply:
As we discussed, this approach will allow us to support multiple file specs safely without the risk of putting to much load on Artifactory